lib/package/config.coffee

```diff
@@ -105,6 +106,75 @@ config =
         type: 'boolean'
         default: false
         order: 13
+      formattettingOptions:
+        title: 'Formatting options'
+        description: 'Specifies [formatting options for `JuliaFormatter.format_text`](https://domluna.github.io/JuliaFormatter.jl/stable/#Formatting-Options-1).'
+        type: 'object'
+        order: 14
+        collapsed: true
+        properties:
+          indent:
```

Comment:
I don't think we need that option -- it's intuitive to have JuliaFormatter follow the current editor, no?

Reply:
It's very intuitive that `indent` and `margin` follow the current editor's ones, but why I added them is that "some" project may want members to have the same setting of `indent` and `margin` for formatting within a project, in the future.
Those `indent` and `margin` may be different from those an user have specified as their general (or language specific) setting.
What I'm thinking about is somewhat will be enabled by https://github.com/domluna/JuliaFormatter.jl/issues/184.

***

But at the same time, I feel those settings are kinda too much for now -- I don't think currently there is a Julia project that encourages members to use the exactly same setting even for local formatting task.

So as a conclusion I'm totally okay to remove these configs for now.

Reply:
Let's keep this for now then.